### ISSUE TITLE
ESM3 from embedding to sequence

### ISSUE THREAD

**Issue by mqyii** [evolutionaryscale/esm#211]
Hi,
Thank you for your model.
I have a quention: I ues ESM3 to get peptide's embedding, and then, I want to map the embedding through the language modeling head to get the sequence logits, and the get the sequence. How to achieve? Or is there any better idea?
